PlayCover/en.lproj/Localizable.strings

```diff
@@ -1,114 +1,93 @@
+"jbBypass.enable" = "Enable Jailbreak Bypass (Alpha)";
+"jbBypass.enable.info" = "Attempts to bypass Jailbreak Detection in some games (No guarentee to work all the time)";
+"displayRefreshRate" = "Screen refresh rate";
+"mouseSensitivity" = "Mouse sensitivity: ";
+"settings.reset" = "Reset settings and keymapping";
+"settings.resetted" = "Settings have been reverted back to default!";
+"setupView.header" = "Are apps not allowing you to login?";
+"setupView.subheader" = "Follow the instructions below to fix exactly that!";
+"setupView.1" = "1) Shutdown your Mac";
+"setupView.2" = "2) Wait for at least 20 seconds for Mac to fully shutdown, then hold the power button";
+"setupView.3" = "3) When \"Loading Startup Options\" appears below the Apple logo, release the power button";
+"setupView.4" = "4) Click the Recovery Mode boot option";
+"setupView.5" = "5) In Recovery, open Terminal (Menu bar -> Utilities -> Terminal)";
+"setupView.6" = "6) Type: ";
+"setupView.7" = "7) Enter the command. Input your Mac login password (The text will be hidden)";
+"setupView.8" = "8) Click the Apple symbol in the Menu bar and Restart";
+"setupView.9" = "If you encountered any errors, repeat the whole process again";
+"setupView.vidTutorials" = "Video Tutorials";
+"setupView.youtube" = "YouTube";
+"setupView.bilibili" = "BiliBili";
+"setupView.pressButtonAndEnterPassword" = "Please disable SIP from below and enter your Mac login password.";
+"setupView.pressButtonAndEnterPassword.button" = "Disable SIP";
+"app.install.unzip" = "Extracting app from ipa file...";
+"app.install.createWrapper" = "Creating app wrapper...";
+"app.install.installPlayTools" = "Installing PlayTools...";
+"app.install.signing" = "Signing app...";
+"app.install.addToLib" = "Adding app to library...";
+"app.install.copy" = "Copying app...";
+"app.install.finished" = "Finished";
+"alert.error" = "Some error happened!";
+"alert.success" = "Success!";
+"notification.appInstalled" = "App installed!";
+"notification.appInstalled.message" = "Check it out now!";
+"setup.enterLoginPassword" = "Please enter your user password";
+"setup.incorrectPassword" = "Password incorrect. Please try again";
+"setup.enterLoginPassword.info" = "Enter the password used to unlock your Mac.";
+"setup.enterLoginPassword" = "Please restart your Mac for changes to take effect.";
+"bottomBar.notices.help" = "News and announcements from the developers";
+"debug.crash" = "Crash app";
+"menubar.log.copy" = "Copy Logs";
+"menubar.documentation" = "Documentation";
+"menubar.website" = "Website";
+"menubar.github" = "GitHub";
+"menubar.discord" = "Discord";
+"menubar.showAppLinks" = "Show app links";
+"xcode.needToInstall" = "You need to install Xcode Commandline tools and restart this App.";
+"button.Install" = "Install";
+"alert.xcodeToolsInstallSucceeded" = "Xcode tools installation succeeded";
+"failedToFetchNotices" = "Failed to find notices. Are we not online?";
```

Comment:
'Are we not online?' is too casual should be 'Please check your internet connection' or something.